sphinxcontrib/jupyter/__init__.py

```diff
@@ -11,6 +11,8 @@ def setup(app):
     app.add_config_value("jupyter_options", None, "jupyter")
     app.add_config_value("jupyter_default_lang", "python3", "jupyter")
     app.add_config_value("jupyter_drop_solutions", False, "jupyter")
+    app.add_config_value("jupyter_drop_tests", False, "jupyter")
```

Comment:
@arnavs I think the default value should be `True`

Reply:
OK, no problem. I'll change it.

Reply:
I should update that for `solutions` also.